src/Ui/UiRequest.py

```diff
@@ -285,24 +299,41 @@ def sendHeader(self, status=200, content_type="text/html", noscript=False, allow
             headers["Access-Control-Allow-Headers"] = "Origin, X-Requested-With, Content-Type, Accept, Cookie, Range"
             headers["Access-Control-Allow-Credentials"] = "true"
 
-        if content_type == "text/html":
-            content_type = "text/html; charset=utf-8"
-        if content_type == "text/plain":
-            content_type = "text/plain; charset=utf-8"
-
         # Download instead of display file types that can be dangerous
         if re.findall("/svg|/xml|/x-shockwave-flash|/pdf", content_type):
             headers["Content-Disposition"] = "attachment"
-
-        cacheable_type = (
-            content_type == "text/css" or content_type.startswith("image") or content_type.startswith("video") or
-            self.env["REQUEST_METHOD"] == "OPTIONS" or content_type == "application/javascript"
+            
+        # UTF-8 character encoding
+        if content_type in ("text/plain", "text/html", "text/css", "application/javascript", "application/json", "application/manifest+json"):
+            content_type = "%s;" % content_type + " " + "charset=utf-8"
```

Comment:
What magic is this??

Reply:
This magic works. If you now start arguing about UTF8 character encoding... What you don't like?

Reply:
Formatting and concatenation in the same expression, lol.

Reply:
Anything is possible in Python. :rofl:

Reply:
Just use `content_type = content_type + " " + "charset=utf-8;"`. This current version is really weird...

Reply:
**text/css; charset=utf-8** :heavy_check_mark: 
~~text/html charset=utf-8;~~

Reply:
**text/html; charset=utf-8** :heavy_check_mark: 
~~text/html charset=utf-8;~~

Reply:
I misplaced `;`. I mean `content_type = content_type + "; " + "charset=utf-8"`.